tests/util/resumeUtils.spec.ts

```diff
@@ -50,6 +50,6 @@ describe('dateInputToDecoratedString', () => {
   });
 
   it('returns a decorated date string if the date input is correct', () => {
-    expect(dateInputToDecoratedString('2000-01', 'placeholder')).toBe('January 2000');
+    expect(dateInputToDecoratedString('2000-01', 'placeholder')).toBe('2000');
```

Comment:
decorated string passed in here

Reply:
Oops, that's acutally the test that tests the `dateInputToDecoratedString` so this is actually correct.